extensions/applicationinsights-clickanalytics-js/src/ClickAnalyticsPlugin.ts

```diff
@@ -23,16 +21,17 @@ import {
 import { PageAction } from './events/PageAction';
 import { AutoCaptureHandler } from "./handlers/AutoCaptureHandler";
 import { DomContentHandler } from "./handlers/DomContentHandler";
+import * as properties from "@microsoft/applicationinsights-properties-js";
```

Comment:
Can we please not do this as it causes minification issues.
Import each individual required value.